### PR TITLE
Make configurable text on Partner’s page

### PR DESCRIPTION
**Pull request by RanaWaqar7550**
### Description

[LP-2405](https://philanthropyu.atlassian.net/browse/LP-2405)

[philu-edx-theme PR](https://github.com/philanthropy-u/philu-edx-theme/pull/1428)

### Notes
Customise partners dashboard page.

### CODE REVIEW

File: openedx/features/partners/models.py [hunk 1]
```diff
@@ -26,6 +26,22 @@ class Partner(TimeStampedModel):
     slug = models.CharField(max_length=100, unique=True, help_text="A unique identifier for an organization")
     email = models.EmailField(help_text="Contact email of an organization")
     configuration = JSONField(null=False, blank=True, default=dumps({"USER_LIMIT": ""}))
+    main_title = models.CharField(blank=True, max_length=100, help_text="Display on partners landing page as main title.")
+    main_description = models.TextField(blank=True, help_text='Brief description of partners landing page.')
```
Comment (qasimgulzar) [inline openedx/features/partners/models.py:30]: please be consistent with the use of ' or "

**Review by aliassad — APPROVED** [review]
LGTM

File: openedx/features/partners/models.py (hunk 1 above)
Comment (RanaWaqar7550) [reply openedx/features/partners/models.py]: fix

**Review by qasimgulzar — APPROVED** [review]
LGTM

**Pull request merged**